types/cache-manager-memcached-store/package.json

```diff
@@ -6,7 +6,7 @@
         "https://github.com/theogravity/node-cache-manager-memcached-store"
     ],
     "dependencies": {
-        "@types/cache-manager": "*"
+        "@types/cache-manager": "3.4.0"
```

Comment:
Not just `<4`?

Reply:
That's the version I see in https://www.npmjs.com/package/cache-manager-memcached-store/v/3.0.0?activeTab=code.

Or do you think it should just be changed to `<4`?

Reply:
You're right, I was looking at v6. But, I would probably still set `<4`.

Reply:
Done.